app/controllers/charges_controller.rb

```diff
@@ -42,7 +43,7 @@ def save_donation_from(user, params)
 
     card_token = params[:stripeToken]
     # it's the stripeToken that we added in the hidden input
-    format.html { redirect_to billing_path, error: 'Oops' } if card_token.nil?
+    redirect_to billing_path, error: 'Oops' if card_token.nil?
```

Comment:
@castrolem let's add error tracking here. Also, change 'Oops' to be `We couldn't process your payment, please try again or contact us at admin@debtcollective.org for support`